cmd/kyma/alpha/create/module/moduleconfig_test.go

```diff
@@ -224,3 +225,73 @@ func TestValidateChannel(t *testing.T) {
 		)
 	}
 }
+
+func TestValidateCustomStateChecks(t *testing.T) {
+	tests := []struct {
+		name    string
+		config  Config
+		wantErr bool
+	}{
+		{
+			name: "No error when custom state check not provided",
+			config: Config{
+				CustomStateChecks: nil,
+			},
+			wantErr: false,
+		},
+		{
+			name: "No error when proper custom state check provided",
+			config: Config{
+				CustomStateChecks: []v1beta2.CustomStateCheck{
+					{
+						JSONPath:    "status.health",
+						Value:       "green",
+						MappedState: v1beta2.StateReady,
+					},
+				},
+			},
+			wantErr: false,
```

Comment:
This should have error returned, a proper custom state should have at least one `Ready` and one `Error` mapped.